src/blocks/accordion/accordion-item/edit.js

```diff
@@ -26,6 +27,7 @@ const TEMPLATE = [
 	[ 'core/paragraph', { placeholder: __( 'Add content…', 'coblocks' ) } ],
 ];
 
+import '../styles/style.scss';
```

Comment:
We should not need to import front-end styles explicitly here. All editor styles should load for this component. Those styles can be found within files that have an editor.scss file.

https://github.com/godaddy-wordpress/coblocks/blob/7769d44feb2a3f53c3d0df5bdf91564813ea8468/src/blocks/accordion/accordion-item/styles/editor.scss#L1